mmdet/models/dense_heads/anchor_head.py

```diff
@@ -52,7 +54,6 @@ def __init__(self,
                      loss_weight=1.0),
                  loss_bbox=dict(
                      type='SmoothL1Loss', beta=1.0 / 9.0, loss_weight=1.0),
-                 loss_normalizer=-1,
                  loss_normalizer_momentum=0.9,
```

Comment:
The default value of `loss_normalizer_momentum` should be 0.0, so that the default setting is still using the real-time `num_total_samples`